labs/fabric-chaincode-wasm.md

```diff
@@ -0,0 +1,26 @@
+# Lab Name
+Hyperledger Fabric WebAssembly chaincode
+
+# Short Description
+The project aims to integrate WebAssembly in hyperledger fabric, which will give chaincode developers a wider variety of languages to choose from when constructing their smart contracts.
+ - Enables user to write WASM function as smart contracts
+
+# Scope of Lab
+This lab will be focused on the integration layer that is required to enable execution of WebAssembly functions in Hyperledger Fabric Smart Contracts. The main piece of the integration will be the WASM chaincode (WASMCC), which will be compatible with the Hyperledger Go Chaincode.
+
+
+More information [here](https://wiki.hyperledger.org/display/INTERN/Running+Web+Assembly+Smart+Contracts+in+Fabric)
+# Initial Committers
+Github IDs for the set of initial committers.
+- [MHBauer](https://github.com/MHBauer) - Morgan Bauer
+- [guoger](https://github.com/guoger) - Jay Guo
+- [swetharepakula](https://github.com/swetharepakula) - Swetha Repakula
+- [kleash](https://github.com/kleash) - Shubham Aggarwal
+
+# Sponsors
```

Comment:
@swetharepakula @guoger @MHBauer : Can you confirm your sponsorship?

Reply:
Confirming

Reply:
Yes.

Reply:
Thanks for confirming.

Reply:
confirm!